internal/controllers/setup.go

```diff
@@ -82,23 +81,39 @@ func setupManager(ctx context.Context, cancel context.CancelFunc, mgr controller
 		)
 	}
 
-	var requiredCrds []string
+	infrastructureTopology := v1.HighlyAvailableTopologyMode
+	if runningOnOpenShift {
+		infrastructureTopology, err = env.GetInfrastructureTopology(ctx, apiReader)
+		if err != nil {
+			return nil, fmt.Errorf("failed to get infrastructure topology: %w", err)
+		}
+	}
 
-	for i := range sspOperands {
-		requiredCrds = append(requiredCrds, getRequiredCrds(sspOperands[i])...)
+	serviceController, err := CreateServiceController()
+	if err != nil {
+		return nil, fmt.Errorf("failed to create service controller: %w", err)
 	}
 
-	// Add VMController necessary VirtualMachine CRD
-	vmKind := strings.ToLower(kubevirtv1.VirtualMachineGroupVersionKind.Kind) + "s"
-	vmCRD := vmKind + "." + kubevirtv1.VirtualMachineGroupVersionKind.Group
-	requiredCrds = append(requiredCrds, vmCRD)
+	return []Controller{
+		serviceController,
+		NewWebhookConfigurationController(),
+		CreateVmController(),
```

Comment:
`CreateVmController` -> `NewVmController` IMO is a nice refactoring to be consistent. (maybe also in a follow up)

Reply:
done